### ISSUE TITLE
[ContextualMenu with section] Screen reader is not narrating the position of menu items

### ISSUE THREAD

**Issue by PraleshGitHub** [microsoft/fluentui#13202]
Environment:

Browser Version: Version 80.0.361.111 (Official build) (64-bit bit)
Windows Version: Windows10
Chrome Version 80.0.3987.163 (Official Build) (64-bit)

Reproduction steps:
1. Open Url https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu
2. Go to ContextualMenu with section headers
3. Export to codepen
4. Open Narrator or NVDA

User Experience:
Screen reader user won’t get proper information about Menu Items like which menu items are related to which sections

Actual:
Narrator doesn’t announce menu items positioning 

Expected:
Narrator should announce the Position for particular section like 1/3 2/3 and 3/3 etc...and not as whole if multiple sections are there in the menu

**Comment by aneeshack4** [comment]
This works on Edge Chromium with Narrator - reading out all the headers in the menu properly. 
![image](https://user-images.githubusercontent.com/4161791/82231719-2d238980-98f3-11ea-8cf7-efad477b971a.png)
Unless you see a gap in the control's accessibility hookups wrt it following the ARIA spec, I don't see how this is actionable to us. This is likely a Chrome specific issue on that browser's end.
